### DISCUSSION TITLE
Got "The template ID is invalid" error after created a new template

### DISCUSSION THREAD

**Discussion by veuler** [emailjs-com/help#78] [Q&A] [answered]
I just have created a new template, even if the template_id is correct, I got this error in a React project. template id's are same as you can see in the below picture.

![image](https://github.com/emailjs-com/help/assets/10335313/9fda010f-22b0-4395-8b46-37f30cfbbb32)

Why I got this error and how can I fix it?

**Comment by xr0master** [comment, marked as answer]
Hi, please be sure you are using the correct public key since the public key and template ID must be from the same account.
